### PR TITLE
fastbit 2.0.2

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

**Comment by dunn** [comment]
So the ML fix should be:
```ruby
  fails_with :clang do
    build 503
    cause "clang++ not able to compile a sample code with std::unordered_map using c++11 or c++0x"
  end
```
Or do I need `if MacOS.version <= :mountain_lion` as well?

**Comment by mikemcquaid** [comment]
The `build` will scope it to the right version, thanks!

**Comment by dunn** [comment]
:weary: 
```
configure: error: llvm-g++ not able to compile a sample code with std::unordered_map using c++11 or c++0x.  Consider using GCC 4.8 or newer
```
Would that be llvm `2336`?

**Comment by mikemcquaid** [comment]
@dunn Yeh, I guess. You may be able to just use `needs :cxx11` as a shortcut if that's the case.

**Comment by dunn** [comment]
Sorry, do you mean `needs :cxx11` would fix both build failures?

**Comment by mikemcquaid** [comment]
Yep, probably, sorry!

File: Library/Formula/fastbit.rb [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
                           "--prefix=#{prefix}"
-    system "make install"
+    system "make", "install"
+    libexec.install lib/"fastbitjni.jar"
+    bin.write_jar_script libexec/"fastbitjni.jar", "fastbitjni"
   end
 
   test do
-    system "#{bin}/fastbit-config", "--version"
+    assert_equal prefix.to_s,
```
Comment (asparagui) [inline Library/Formula/fastbit.rb]: :trophy:

**Comment by asparagui** [comment]
Thanks!

**Pull request closed without merging**